sdk/communication/azure-communication-chat/samples/chat_thread_client_sample_async.py

```diff
@@ -132,7 +132,8 @@ async def send_message_async(self):
                 # Scenario 1: Send message without specifying chat_message_type
                 send_message_result = await chat_thread_client.send_message(
                     "Hello! My name is Fred Flinstone",
-                    sender_display_name="Fred Flinstone")
+                    sender_display_name="Fred Flinstone",
+                    metadata={"tags": "tags"})
```

Comment:
As part of the sample, could we show an example of how to consume the metadata tags as well?

For example, you could replace line 148 with the following

```python
chat_message_1 = await chat_thread_client.get_message(send_message_result_id)
print("Message obtained contains content: ", chat_message_1.content.message, " with metadata: ", chat_message_1.metadata)
```